deploy/charts/venafi-kubernetes-agent/templates/configmap.yaml

```diff
@@ -191,4 +191,11 @@ data:
           group: jetstack.io
           version: v1alpha1
           resource: venafiissuers
+    - kind: "k8s-dynamic"
+      name: "k8s/fireflyissuers"
+      config:
+        resource-type:
+          group: firefly.venafi.com
+          version: v1
+          resource: firefly
```

Comment:
Suggestion(blocking): The resource is `issuer`

Reply:
Changed it to `issuers` - **plural**.